lib/src/common/js_proxy.dart

```diff
@@ -145,11 +145,32 @@ dynamic dartValue(jsValue) {
       }
       _addDartInstance(jsValue, dartMap);
       return dartMap;
+    } else if (jsValue is JsFunction) {
+      return _dartType(jsValue);
```

Comment:
Since this is very generic its pretty hard to tell what the purpose might be. One possibly common case would be if you had a dart class with a method that takes a closure as an argument, and you wanted that to be usable from JS.